handlers/high-charts/src/utils.ts

```diff
@@ -88,3 +88,35 @@ export async function sendOSC(jsonFile: string, outFile: string, server: string,
         })
     ]);
 }
+
+/**
+ * The function will return the string representing the graph title and axes information
+ * @param highChartsData 
+ * @returns graphInfo string
+ */
+export function getGraphInfo(highChartsData: any): string{
```

Comment:
Not a good idea to use `any` generally. But since we don't have types to go off of it's probably fine for now.